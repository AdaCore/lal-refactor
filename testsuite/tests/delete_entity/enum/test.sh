cp ./main.orig ./main.adb
delete_entity -P default.gpr -S main.adb -L 2 -R 21
echo "--  (main.orig) -L 2 -R 21"
cat ./main.adb
