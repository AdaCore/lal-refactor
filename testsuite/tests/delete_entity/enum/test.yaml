title: delete_enum

description: delete_enity tool test (enumeration literal)

driver: shell_script
