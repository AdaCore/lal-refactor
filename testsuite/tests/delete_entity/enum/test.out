--  (main.orig) -L 2 -R 21
procedure Main is
   type Enum is (A, C);

   Map : array (C .. C) of Integer :=
     (C => 3);

   subtype S1 is Enum range A .. A;
   subtype S2 is Enum range C .. C;
   subtype S3 is Enum range C .. A;
begin
   null;
end;
